### PR TITLE
PAPP-23169  ThreatStream: Release notes and MR conflicts

### PR DESCRIPTION
**Pull request by cpangam**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [ ] Testing of all the changes has been performed (for bug fixes / features)
- [ ] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
- [ ] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

**Review by mpan-splunk — DISMISSED** [review]

File: threatstream_connector.py [hunk 1]
```diff
@@ -635,6 +635,7 @@ def _file_reputation(self, param):
     def _domain_reputation(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
         value = self._handle_py_ver_compat_for_input_str(param[THREATSTREAM_JSON_DOMAIN])
+        extend_source = param.get("extend_source", False)
```
Comment (paulnawat) [inline threatstream_connector.py:638]: I believe `extend_souce` is not being used here, right?

File: threatstream_connector.py [hunk 2]
```diff
@@ -652,6 +654,7 @@ def _domain_reputation(self, param):
     def _ip_reputation(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
         value = param[THREATSTREAM_JSON_IP]
+        extend_source = param.get("extend_source", False)
```
Comment (paulnawat) [inline threatstream_connector.py:657]: I believe the extend_source is not being used (see https://github.com/splunk-soar-connectors/threatstream/blob/f4639f71065827c21d329dab883d25b494018b01/threatstream_connector.py#L657)

**Review by paulnawat — APPROVED** [review]
lgtm, thanks!

File: release_notes/release_notes.html [hunk 3]
```diff
@@ -1,5 +1,26 @@
 <b>ThreatStream Release Notes - Published by Splunk June 15, 2021</b>
 <br><br>
+<b>Version 3.2.8 - Released November 29, 2021</b>
```
Comment (paulnawat) [inline release_notes/release_notes.html:3]: @cpangam Could you please confirm if we also need to include release notes for the 3 changes from the Epic ticket PAPP-20942?
The changes are from PAPP-19922, PAPP-21485, and PAPP-21096.

Comment (cpangam) [reply release_notes/release_notes.html:3]: Yes. That was needed. These are previous release notes and not included in the release_notes.html

**Pull request merged**